### PR TITLE
Changes to bucket endpoint config

### PR DESCRIPTION
**Pull request by MancunianSam**
This updates the TDR request handler to be almost identical to the AWS one but with `credentials: "include"` set. It also updates the way we do the S3 config so that it sends the request to the cloudfront distribution.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (TomJKing) [inline package-lock.json:1]: Is this supposed to be here? Appear to have two `package-lock.json` files now?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one comment.

File: package-lock.json (hunk 1 above)
Comment (MancunianSam) [reply package-lock.json]: No, I must have run npm commands in this directory by mistake, I've deleted it.

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]

**Pull request merged**